products.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once 'src/Factories/PdoFactory.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Services/ProductListDisplayService.php';
+require_once 'src/Models/CategoryModel.php';
+require_once 'src/Entities/Product.php';
+
+$db = PdoFactory::connect();
+
+$productList = ProductModel::getProductList($_GET['cat_id'], $db);
```

Comment:
what happens if there is no cat_id? or if its a wrd instead of an id? need some validation

Reply:
We have inserted 'invalid ID' if statement for this.